src/utils/date-service.ts

```diff
@@ -0,0 +1,5 @@
+import { differenceInSeconds } from 'date-fns';
+
+export const getTimeTrackedInSeconds = (startTime: Date, endTime: Date | null): number => {
```

Comment:
Please, use destructuring so we don't have to care about parameter order.

Reply:
fixed